src/components/activity-completion/completion-page-content.tsx

```diff
@@ -1,50 +1,165 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import IconCheck from "../../assets/svg-icons/icon-check.svg";
 import { showReport } from "../../utilities/report-utils";
+import { Sequence, Activity, EmbeddableWrapper } from "../../types";
+import { renderHTML } from "../../utilities/render-html";
+import { watchAllAnswers } from "../../firebase-db";
+import { isQuestion } from "../../utilities/activity-utils";
+import { refIdToAnswersQuestionId } from "../../utilities/embeddable-utils";
+import ccPlaceholderLogo from "../../assets/cc-placeholder.png";
 
 import "./completion-page-content.scss";
 
 interface IProps {
+  activity: Activity;
   activityName: string;
-  isActivityComplete: boolean;
   onPageChange: (page: number) => void;
   showStudentReport: boolean;
   thumbnailURL: string | null;
+  onOpenReport?: () => void;
+  sequence?: Sequence;
+  activityIndex?: number;
+  onActivityChange?: (activityNum: number) => void;
+  onShowSequence?: () => void;
 }
 
 export const CompletionPageContent: React.FC<IProps> = (props) => {
-  const { activityName, isActivityComplete, onPageChange, showStudentReport, thumbnailURL } = props;
+  const { activity, activityName, onPageChange, showStudentReport, thumbnailURL,
+    sequence, activityIndex, onActivityChange, onShowSequence } = props;
+  
+  const [answers, setAnswers] = useState<any>();
+  let activityCompletionArray: boolean[]=[];
```

Comment:
spacing:
`let activityCompletionArray: boolean[] = [];`

Reply:
Also I would scope this inside of the function that uses it.  Doesn't seem to be a need to declare it here.